app/uk/gov/hmrc/gform/views/form/snippets/field_template_text.scala.html

```diff
@@ -26,14 +26,25 @@
 @import uk.gov.hmrc.gform.views.summary.TextFormatter
 
 @import uk.gov.hmrc.gform.validation.{FormFieldValidationResult, ValidationUtil}
+@import akka.japi.Option.Some
 @(fieldValue: FormComponent, text: Text, prepop: String, validationResult: Option[FormFieldValidationResult], index: Int, sectionTitle: String)
 
 @hasErrors = @{
   validationResult.exists(_.isNotOk)
 }
+@inputBoxSize = @{
+    fieldValue.`type` match{
+        case t@Text(_, _) => {
+            t.constraint match{
+                case TextWithRestrictions(_, (max)) if(max<5) => "input--xsmall"
```

Comment:
Three things on this line:

1. Nested pattern matching can be avoided.
2. Redundant parentheses around `max`
3. Redundant parentheses around `max<5`